text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndExport.scala

```diff
@@ -50,7 +54,10 @@ object ExtractAndExport extends App {
     // 3. Extract causal mentions from the texts
     // todo: here I am choosing to pass each text/section through separately -- this may result in a difficult coref problem
     val mentions = texts.flatMap(reader.extractFromText(_, filename = Some(file.getName)))
+    //The version of mention that includes routing between text vs. comment
+//    val mentions = texts.flatMap(text => textRouter.route(text).extractFromText(text, filename = Some(file.getName))).seq
```

Comment:
@BeckySharp, changed the extract and align a bit to also have the version with routing between text and comment-like text